src/lib/sw_engine/tvgSwRaster.cpp

```diff
@@ -213,14 +213,42 @@ static bool _translucentRle(SwSurface* surface, const SwRleData* rle, uint32_t c
     auto span = rle->spans;
     uint32_t src;
 
+#if defined(THORVG_NEON_VECTOR_SUPPORT)
+    uint8x8_t vSrc;
+    uint8x8_t vCol = (uint8x8_t) vdup_n_u32(color);
+#endif
+
     for (uint32_t i = 0; i < rle->size; ++i) {
         auto dst = &surface->buffer[span->y * surface->stride + span->x];
+
+#if defined(THORVG_NEON_VECTOR_SUPPORT)
+        uint8x8_t *vDst = (uint8x8_t*) dst;
+#endif
+
         if (span->coverage < 255) src = ALPHA_BLEND(color, span->coverage);
         else src = color;
         auto ialpha = 255 - surface->blender.alpha(src);
+
+#if defined(THORVG_NEON_VECTOR_SUPPORT)
+        uint8x8_t vSrc = (uint8x8_t) vdup_n_u32(src);
+        uint8x8_t vIalpha = (uint8x8_t) vdup_n_u32(ialpha);
+
+        uint32_t iterations = span->len / 2;
+        uint32_t left = span->len % 2;
+
+        for (uint32_t x = 0; x < iterations; x+=2) {
```

Comment:
Please validate `x+=2` is correct

Reply:
Yes it is. vSrc contains data from two pixels (2x uint32_t)